packages/openapi-ts/src/compiler/typedef.ts

```diff
@@ -123,7 +123,15 @@ export const createTypeRecordNode = (
 ) => {
     const keyNode = createTypeUnionNode(keys);
     const valueNode = createTypeUnionNode(values);
-    const node = ts.factory.createTypeReferenceNode('Record', [keyNode, valueNode]);
+    // NOTE: We use the syntax `{ [key: string]: string }` because using a Record causes
+    //       invalid types with circular dependencies. This is functionally the same.
```

Comment:
Can you link the issue in comments please

Reply:
done